back/src/Guide/Database/Utils.hs

```diff
@@ -118,7 +118,7 @@ queryRowMaybe = QuasiQuoter
           (#prepared False)
           (#params toPostgresParams)
           (#result (HD.rowMaybe fromPostgresRow))
-          $(stringE sql)|]
+          (fromString $(stringE sql))|]
```

Comment:
This is needed so that `-XOverloadedStrings` would not be necessary to enable at the use site.